src/main/java/ExpiredTokenJob.java

```diff
@@ -0,0 +1,53 @@
+import fr.maif.socleapi.croupier.model.TokenInfos;
```

Comment:
The import 'fr.maif.socleapi.croupier.model.TokenInfos' seems to be from an external library. Make sure to handle dependencies properly.